src/core/fetchRequest.js

```diff
@@ -90,12 +90,12 @@ const requestFactory = (url, options) => {
             return response.json().catch(() => ({}));
         })
         .then(response => {
-            if (responseCode === 204 || responseCode === 202) {
+            if (responseCode === 204) {
                 return null;
             }
 
             // successful request
-            if (responseCode === 200 || response.success === true) {
+            if (responseCode >= 200 || responseCode <= 299 || response.success === true) {
```

Comment:
It looks like condition is wrong since `responseCode >= 200` returns true for response status 404

Reply:
I fixed the operator, that was a wrong commit. I also decreased the limit to 266 because that's the last request in 2xx